test/unit/cpu/test-indexsets.cxx

```diff
@@ -20,6 +23,14 @@ class IndexSetTest : public ::testing::Test
   RAJA::IndexSet index_sets_[NumBuildMethods];
 };
 
+template <typename T>
+class DependentIndexSetTest : public ::testing::Test
```

Comment:
Should we move this part down by the tests its actually associated with? It looks like there are some tests in between the class and the associated tests.